services/agent_services/service_configs/dff_knowledge_prompted_skill/service.yml

```diff
@@ -0,0 +1,17 @@
+name: agent
+endpoints:
+- respond
+compose:
+  command: sh -c 'bin/wait && python -m deeppavlov_agent.run agent.pipeline_config=assistant_dists/dream_kg_prompted/pipeline_conf.json'
```

Comment:
the file's name is incorrect

Reply:
Removed. Cards already exist for dream_kg_prompted